cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/InMemoryStreamCoordinatorTest.java

```diff
@@ -34,21 +44,42 @@
 
   private static Injector injector;
 
+  private static TransactionManager transactionManager;
+  private static DatasetService datasetService;
+
   @BeforeClass
   public static void init() {
     injector = Guice.createInjector(
       new ConfigModule(),
       new DiscoveryRuntimeModule().getInMemoryModules(),
       new DataFabricModules().getInMemoryModules(),
+      new DataSetServiceModules().getInMemoryModule(),
```

Comment:
Looks like for many of the existing tests, all you need is a dummy implementation of `StreamMetaStore.class` instead of injecting a real one, which requires a lot of changes in dependency.